Update resource section (#2415)
altair-viz/altair: doc/user_guide/ecosystem.rst

.. List of links.
.. _`Data Visualization Course`: https://viz-learn.mds.ubc.ca
.. _`Key Capabilities for Data Science`: https://extendedlearning.ubc.ca/programs/key-capabilities-data-science

`Brief Introduction Videos`_
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Calmcode provides a few videos that give a brief overview of the Altair library.

.. List of links.
.. _`Brief Introduction Videos`: https://calmcode.io/altair/introduction.html

.. _altair-ecosystem:

Altair Ecosystem
----------------

This is a list of projects directs related to Altair. There are many other packages that can be used in tandem with Altair, e.g. `dashboard packages which you can read more about in the answers to this StackOverflow question`_.

.. List of links.
.. _`dashboard packages which you can read more about in the answers to this StackOverflow question`: https://stackoverflow.com/questions/49833866/making-dashboards-using-altair

Vega-Lite_
~~~~~~~~~~

The higher-level visualization grammar that Altair implements in Python.

.. List of links.

> .. _Vega-Lite: https://vega.github.io/vega-lite
> 
> altair_saver_

Reviewer: ```suggestion
.. _Vega-Lite: https://vega.github.io/vega-lite

vl-convert_
~~~~~~~~~~~

Python library for converting Altair/Vega-Lite chart specifications into static images (SVG or PNG) or Vega chart specifications without any external dependencies.

.. List of links.
.. _vl-convert: https://github.com/vega/vl-convert

altair_saver_
```

@joelostblom Happy to review this PR if you want

Author: @binste Yes that would be great, thank you!

.. List of links.
.. _`Data Visualization Course`: https://viz-learn.mds.ubc.ca
.. _`Key Capabilities for Data Science`: https://extendedlearning.ubc.ca/programs/key-capabilities-data-science

`Brief Introduction Videos`_
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Calmcode provides a few videos that give a brief overview of the Altair library.

.. List of links.
.. _`Brief Introduction Videos`: https://calmcode.io/altair/introduction.html

.. _altair-ecosystem:

Altair Ecosystem
----------------

This is a list of projects directs related to Altair. There are many other packages that can be used in tandem with Altair, e.g. `dashboard packages which you can read more about in the answers to this StackOverflow question`_.

.. List of links.
.. _`dashboard packages which you can read more about in the answers to this StackOverflow question`: https://stackoverflow.com/questions/49833866/making-dashboards-using-altair

Vega-Lite_
~~~~~~~~~~

The higher-level visualization grammar that Altair implements in Python.

.. List of links.
.. _Vega-Lite: https://vega.github.io/vega-lite

vl-convert_
~~~~~~~~~~~

> Python library for converting Altair/Vega-Lite chart specifications into static images (SVG or PNG) or Vega chart specifications without any external dependencies.
> 
> .. List of links.
> .. _vl-convert: https://github.com/vega/vl-convert
> 

Author: ```suggestion

VegaFusion_
~~~~~~~~~~~

VegaFusion provides server-side scaling for Altair charts, which can accelerate interactive charts, extract transformed data, and perform data-intensive aggregations on the server and prune unused columns from the source datase yielding smaller size visualizations

.. List of links.
.. _VegaFusion: https://vegafusion.io/

```

.. _ecosystem:

Resources & Ecosystem
=====================

We hope to make it easier to find these packages and learning resources by listing them here. If you know of a project that should be added, please let us know.

.. _learning-resources:

Learning Resources
------------------

This is a list of a resources that complements the official documentation and can help you learn more about how to use Altair.

`Visualization Curriculum`_
~~~~~~~~~~~~~~~~~~~~~~~~~~~

A data visualization curriculum from the UW data group that developed Vega-Lite.

.. List of links.
.. _`Visualization Curriculum`: https://uwdata.github.io/visualization-curriculum

`Jupyter Notebook Tutorials`_
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Jupyter Notebooks tutorials and examples from the Altair authors.

.. List of links.
.. _`Jupyter Notebook Tutorials`: https://github.com/altair-viz/altair_notebooks

`Pycon Tutorial`_
~~~~~~~~~~~~~~~~~

> Altair tutorial given at PyCon 2018 by the Altair authors.

Reviewer: ```suggestion
Altair tutorial given at PyCon 2018 by the Altair author Jake VanderPlas.
```
Maybe make it more specific as the recording shows only Jake giving that talk?

.. List of links.
.. _`Data Visualization Course`: https://viz-learn.mds.ubc.ca
.. _`Key Capabilities for Data Science`: https://extendedlearning.ubc.ca/programs/key-capabilities-data-science

`Brief Introduction Videos`_
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Calmcode provides a few videos that give a brief overview of the Altair library.

.. List of links.
.. _`Brief Introduction Videos`: https://calmcode.io/altair/introduction.html

.. _altair-ecosystem:

Altair Ecosystem
----------------

This is a list of projects directs related to Altair. There are many other packages that can be used in tandem with Altair, e.g. `dashboard packages which you can read more about in the answers to this StackOverflow question`_.

.. List of links.
.. _`dashboard packages which you can read more about in the answers to this StackOverflow question`: https://stackoverflow.com/questions/49833866/making-dashboards-using-altair

Vega-Lite_
~~~~~~~~~~

The higher-level visualization grammar that Altair implements in Python.

.. List of links.
.. _Vega-Lite: https://vega.github.io/vega-lite

> vl-convert_
> ~~~~~~~~~~~
> 
> 
> VegaFusion_

Reviewer: Somehow my previous code suggestion wasn't applied completely, only the title, so I pasted it again below.
```suggestion
vl-convert_
~~~~~~~~~~~

Python library for converting Altair/Vega-Lite chart specifications into static images (SVG or PNG) or Vega chart specifications without any external dependencies.

.. List of links.
.. _vl-convert: https://github.com/vega/vl-convert

VegaFusion_
```

.. List of links.
.. _VegaFusion: https://vegafusion.io/

altair_saver_
~~~~~~~~~~~~~

Enables saving charts to a variety of output types.

.. List of links.
.. _altair_saver: https://github.com/altair-viz/altair_saver

altair_data_server_
~~~~~~~~~~~~~~~~~~~

Data transformer plugin that transparently serves data for charts.

.. List of links.
.. _altair_data_server: https://github.com/altair-viz/altair_data_server

altair_pandas_
~~~~~~~~~~~~~~

Altair backend for the pandas plotting API.

.. List of links.
.. _altair_pandas: https://github.com/altair-viz/altair_pandas

vega_datasets_
~~~~~~~~~~~~~~

Offline access to the Vega datasets used in the Altair documentation.

.. List of links.
.. _vega_datasets: https://github.com/altair-viz/vega_datasets

altair_recipes_
~~~~~~~~~~~~~~~

altair_recipes provides a collection of ready-made statistical graphics for Altair. See the `docs <https://altair-recipes.readthedocs.io/en/latest/>`__.

.. List of links.
.. _altair_recipes: https://github.com/piccolbo/altair_recipes

> gpdvega_

Reviewer: Is this package still of use? I'm not familiar with it so maybe I miss something. Last commit on GitHub is from 2018 and it's main feature seems to be support for GeoPandas dataframes which was added to Altair in 3.3.0 (2019). I was able to execute the examples from the documentation of gpdvega without having it installed or importing it.

Can't make a code suggestion here as the diff range is too short.

Author: Good point, I think we can remove it since the Altair docs now nicely shows how to work with geodata

Author: Removed
